docs/developer/explanation/code.rst

```diff
@@ -3,56 +3,32 @@ Launchpad Code Hosting
 
 .. include:: ../../includes/important_not_revised.rst
 
-Launchpad hosts source code using `Git <https://git-scm.com/>`__ and
-`Bazaar <https://www.breezy-vcs.org/>`__ branches. However dealing
-with code in Launchpad covers many more areas than just hosting
+Launchpad hosts source code using `Git <https://git-scm.com/>`__ branches. 
+However dealing with code in Launchpad covers many more areas than just hosting
 branches.
 
 Code for the web application is only part of the Launchpad codehosting
 system. The major sub-systems are:
 
--  The `git` and `bzr` / `brz` clients (neither of which is part of
-   Launchpad, but their behaviours are important to us)
--  Connectivity to Launchpad (git, git+ssh, and https for Git; SFTP and
-   bzr+ssh for Bazaar)
+-  The `git` client (which is not a part of Launchpad but its behaviour is 
+   important to us).
+-  Connectivity to Launchpad (git, git+ssh, and https for Git)
 -  Hosting infrastructure
 -  The underlying object model
 -  The web application
 -  Email processing
--  Code imports (from CVS, Subversion, git and Mercurial)
+-  Code imports from git 
 -  Branch source code browser (`cgit <https://git.zx2c4.com/cgit/>`__
-   for Git; :doc:`loggerhead <../how-to/land-update-for-loggerhead>` for Bazaar)
--  Source package recipes (`git-build-recipe`/`bzr-builder\` integration
-   with :doc:`Soyuz ../how-to/use-soyuz-locally`)
+   for Git)
+-  Source package recipes (`git-build-recipe` integration with 
+   :ref:`Soyuz <use-soyuz-locally>`)
 
 Each of these subsystems also have multiple moving parts and some have
 other asynchronous jobs associated with them.
 
 The `codehosting overview diagram :attachment:../images/codehosting.png`
 summarises how some of these systems interact.
 
-You can :doc:`run the codehosting system locally <../how-to/codehosting-locally>`.
```

Comment:
Should we link the page to code hosting locally with turnip?
https://turnip.readthedocs.io/en/latest/development.html